.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+---
+name: CI
+'on':
+  pull_request:
+  push:
+    branches:
+      - '*'
+  schedule:
+    - cron: "30 5 * * 1"
+
+defaults:
+  run:
+    working-directory: 'theforeman.foreman_scap_client'
+
+jobs:
+
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+        with:
+          path: 'theforeman.foreman_scap_client'
+
+      - name: Set up Python 3.
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install test dependencies.
+        run: pip3 install yamllint ansible-lint
+
+      - name: Lint code.
+        run: |
+          yamllint .
+          ansible-lint
+
+  molecule:
+    name: Molecule
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - distro: centos7
+          - distro: centos8
+
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+        with:
+          path: 'theforeman.foreman_scap_client'
+
+      - name: Set up Python 3.
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install test dependencies.
+        run: pip3 install ansible molecule[docker] docker testinfra
```

Comment:
The reason for not using requirements.txt is that we only need specific dependencies and not all of them for given step, correct?

Reply:
I've copied this pretty much directly from [geerlingguy/ansible-role-drupal](https://github.com/geerlingguy/ansible-role-drupal/blob/master/.github/workflows/ci.yml).
Looks like I didn't end up committing a changed `requirements.txt`.